src/main/java/org/reficio/p2/P2Mojo.java

```diff
@@ -545,14 +548,18 @@ private void executeCategoryPublisher() throws AbstractMojoExecutionException, I
         publisher.execute();
     }
 
+    @SuppressFBWarnings("DM_DEFAULT_ENCODING")
     private void prepareCategoryLocationFile() throws IOException {
         if (categoryFileURL == null || categoryFileURL.trim().isEmpty()) {
-            InputStream is = getClass().getResourceAsStream(DEFAULT_CATEGORY_CLASSPATH_LOCATION + DEFAULT_CATEGORY_FILE);
-            File destinationFolder = new File(destinationDirectory);
-            destinationFolder.mkdirs();
-            File categoryDefinitionFile = new File(destinationFolder, DEFAULT_CATEGORY_FILE);
-            FileWriter writer = new FileWriter(categoryDefinitionFile);
-            IOUtils.copy(is, writer, "UTF-8");
+            File categoryDefinitionFile;
+            FileWriter writer;
+            try (InputStream is = getClass().getResourceAsStream(DEFAULT_CATEGORY_CLASSPATH_LOCATION + DEFAULT_CATEGORY_FILE)) {
+                File destinationFolder = new File(destinationDirectory);
```

Comment:
Any reason why we do not want to define an encoding explicitly? System.getDefaultEncoding() would also work for me.

Reply:
I dont know. It is the original implementation. I will run the ITs again with default encoding and we will see what happens

Reply:
I see that IOUtils uses UTF-8 explicitly. I think you should just use UTF-8 then.